package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mercurius",
-  "version": "11.1.0",
+  "version": "11.2.0",
```

Comment:
please revert this

Reply:
Is there a rule on when the package version gets bumped in this repo?

Reply:
the releaser bumps the version when they release.